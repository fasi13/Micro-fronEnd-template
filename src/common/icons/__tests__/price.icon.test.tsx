import { render } from '@testing-library/react';
import React from 'react';
import { PriceIcon } from '../index';

test('It should render priceICon', async () => {
	const size = { width: 512, height: 512 };

	render(<PriceIcon className="" width={size.width} height={size.height} />);
});
