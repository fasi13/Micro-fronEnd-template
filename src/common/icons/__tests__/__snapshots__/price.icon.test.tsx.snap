// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`It should render priceICon 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <svg
        aria-hidden="true"
        class="MuiSvgIcon-root"
        fill="#6D899E"
        focusable="false"
        height="512"
        viewBox="0 0 520 520"
        width="512"
        x="0px"
        xmlns="http://www.w3.org/2000/svg"
        y="0px"
      >
        <g>
          <path
            d="m509.665 2.343c-3.124-3.123-8.189-3.123-11.313 0l-44.302 44.301c-6.324-4.194-13.901-6.644-22.042-6.644h-162.745c-10.686 0-20.73 4.161-28.284 11.716l-230.143 230.142c-14.448 14.45-14.45 37.831 0 52.284l167.029 167.028c14.446 14.448 37.835 14.451 52.285.001 27.856-27.856 223.171-223.171 230.143-230.143 7.555-7.556 11.715-17.601 11.715-28.284v-162.744c0-8.141-2.45-15.717-6.644-22.042l44.301-44.301c3.125-3.125 3.125-8.189 0-11.314zm-53.657 77.657v162.745c0 6.41-2.497 12.438-7.029 16.971l-230.143 230.142c-8.195 8.195-21.461 8.196-29.657-.001l-167.028-167.028c-8.195-8.197-8.198-21.459 0-29.657l230.142-230.143c4.532-4.533 10.559-7.029 16.97-7.029h162.745c3.701 0 7.207.843 10.341 2.345l-38.553 38.553c-31.46-22.423-75.788.011-75.788 39.102 0 26.468 21.533 48 48 48 39.092 0 61.523-44.329 39.102-75.788l38.553-38.553c1.502 3.133 2.345 6.64 2.345 10.341zm-80 88c-17.645 0-32-14.355-32-32 0-24.695 26.987-40.133 48.242-27.556l-21.898 21.898c-3.125 3.125-3.125 8.189 0 11.314 3.125 3.124 8.189 3.123 11.313 0l21.899-21.899c12.564 21.235-2.834 48.243-27.556 48.243z"
          />
        </g>
      </svg>
    </div>
  </body>,
  "container": <div>
    <svg
      aria-hidden="true"
      class="MuiSvgIcon-root"
      fill="#6D899E"
      focusable="false"
      height="512"
      viewBox="0 0 520 520"
      width="512"
      x="0px"
      xmlns="http://www.w3.org/2000/svg"
      y="0px"
    >
      <g>
        <path
          d="m509.665 2.343c-3.124-3.123-8.189-3.123-11.313 0l-44.302 44.301c-6.324-4.194-13.901-6.644-22.042-6.644h-162.745c-10.686 0-20.73 4.161-28.284 11.716l-230.143 230.142c-14.448 14.45-14.45 37.831 0 52.284l167.029 167.028c14.446 14.448 37.835 14.451 52.285.001 27.856-27.856 223.171-223.171 230.143-230.143 7.555-7.556 11.715-17.601 11.715-28.284v-162.744c0-8.141-2.45-15.717-6.644-22.042l44.301-44.301c3.125-3.125 3.125-8.189 0-11.314zm-53.657 77.657v162.745c0 6.41-2.497 12.438-7.029 16.971l-230.143 230.142c-8.195 8.195-21.461 8.196-29.657-.001l-167.028-167.028c-8.195-8.197-8.198-21.459 0-29.657l230.142-230.143c4.532-4.533 10.559-7.029 16.97-7.029h162.745c3.701 0 7.207.843 10.341 2.345l-38.553 38.553c-31.46-22.423-75.788.011-75.788 39.102 0 26.468 21.533 48 48 48 39.092 0 61.523-44.329 39.102-75.788l38.553-38.553c1.502 3.133 2.345 6.64 2.345 10.341zm-80 88c-17.645 0-32-14.355-32-32 0-24.695 26.987-40.133 48.242-27.556l-21.898 21.898c-3.125 3.125-3.125 8.189 0 11.314 3.125 3.124 8.189 3.123 11.313 0l21.899-21.899c12.564 21.235-2.834 48.243-27.556 48.243z"
        />
      </g>
    </svg>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
